app/src/main/res/layout/no_internet.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/no_internet_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingBottom="120dp"
+    android:gravity="center">
+
+    <ImageView
```

Comment:
Content description lint warning on ImageView elements can be suppressed by this line:
`android:contentDescription="@null"`